src/Sylius/Bundle/AddressingBundle/spec/EventListener/ZoneMemberIntegrityListenerSpec.php

```diff
@@ -13,12 +13,14 @@
 
 namespace spec\Sylius\Bundle\AddressingBundle\EventListener;
 
+use InvalidArgumentException;
```

Comment:
Implicit PSR rule (via example in PSR, not a actual rule) and usual sylius way is to use it directly in source as `\InvalidArgumentException::class`